src/StructureMap.DynamicInterception/MethodInvocation.cs

```diff
@@ -39,6 +54,21 @@ public MethodInfo MethodInfo
             get { return _invocation.Method; }
         }
 
+        public MethodInfo MethodInfoInvocationTarget
```

Comment:
Similarly to `TargetType`, `TargetObject`, I would call it `TargetMethodInfo`. Also, could you please come up with any use case when this property might be useful given that we have `MethodInfo` property already?